src/component/calculator.js

```diff
@@ -4,6 +4,7 @@ const Calculator = () => (
   <div className="single-pad">
     <Screen />
   </div>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228775554-19085504-5384-4eab-a5e9-c26d10d6e6f7.png)


- [ ] It would be great if you make all the code in your calculator.js component and just make the screen of the calculator in your Screen.js component to utilize your calculator.js component because now you are having all the code in your Screen component and import it into your calculator.js component 🤔 which is making your calculator.js component useless 😨 kindly have your state in the calculator.js component and pass props to Screen.js component to follow the requirements 😊